index.js

```diff
@@ -65,7 +65,10 @@ var defaults = {
 	variables: {},
 	// Preserve variables injected via JS with the `variables` option above
 	// before serializing to CSS (`false` will remove these variables from output)
-	preserveInjectedVariables: true
+	preserveInjectedVariables: true,
+	// Will write media queries in the same order as in the original file.
+	// Set it to true if you're working with min-width
+	preserveAtRulesOrder: false
```

Comment:
The default is still `false` here when it should be `true`

All of the test cases would need to be updated as well.